### PR TITLE
Release preparations

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
 # git master
 
-* cmake 2.8.9 or later is now required
+# 2015.11
+
+* cmake 2.8.9 or later is now required; reduced cmake 3 warnings
+* cmake 3 is required for installing projects correctly (@tribal-tec
+  correct?)
```
Comment (tribal-tec) [inline CHANGES.md]: ...for finding a project with find_package() from outside the build tree/from the install tree.

File: CHANGES.md [hunk 2]
```diff
@@ -1,6 +1,22 @@
 # git master
```
Comment (tribal-tec) [inline CHANGES.md]: Added AppleCheckOpenGL for foo?

Comment (tribal-tec) [inline CHANGES.md]: Added CommonDate

**Comment by eile** [comment]
all done.

**Comment by tribal-tec** [comment]
+1 for me
@hernando ?

**Pull request merged**